### ISSUE TITLE
Update rule no-unconditional-jump/S1751: ignore 'continue'

### ISSUE THREAD

**Issue by vilchik-elena** [SonarSource/SonarTS#589]
Issue for case like this, should be raised by rule #588 so it should be excluded from [no-unconditional-jump/S1751](https://jira.sonarsource.com/browse/RSPEC-1751). This is because [no-unconditional-jump/S1751](https://jira.sonarsource.com/browse/RSPEC-1751) should raise issues only for loops executed once.
```typescript
while (cond) {
  doSomething();
  continue; // Noncompliant
}
```
